jscripts/thread.js

```diff
@@ -40,7 +40,13 @@ var Thread = {
 		var new_post_ids = new Array();
 		var quoted = Cookie.get("multiquote");
 		var is_new = true;
-		if(quoted)
+		if($("#pid" + pid).next("div.post").hasClass('deleted_post'))
+		{
+			$.jGrowl(lang.post_deleted_error, {theme:'jgrowl_error'});
+			var deleted = true;
+		}
+
+		if(quoted && !deleted)
```

Comment:
From a quick look, this should raise a fatal error if `deleted` is not defined. You should add `var deleted = false` after `is_new = true` and change its value within the deleted_post check above.

Reply:
Done!